src/prompts/alert/index.tsx

```diff
@@ -46,18 +64,26 @@ export class Alert extends Component<PromptUiProps<AlertPromptConfig>, never> {
             color: acceptActionFg
         };
 
+        const iconStyle = {
+            ...styles.iconStyle,
+            backgroundImage: `url(${platformConfig?.iconUrl})`
+        };
+
         return (
             <div style={containerStyle} className={classes}>
-                <div className="kumulos-alert-header">
-                    <h1>{heading}</h1>
-                    <div className="kumulos-alert-icon">
-                        <img src={platformConfig?.iconUrl} />
+                <div
```

Comment:
Not necessary to change, but curious why using a background-image vs an img tag here?

Reply:
Mostly down to dealing with icon sizes a bit better, felt it gave a better appearance leveraging background image positioning/layout than it did with img!

Reply:
Cool, sounds good. Guess unlikely anyone had overridden styling for old markup just yet.